libvast/include/vast/concept/printable/vast/pattern.hpp

```diff
@@ -23,7 +25,9 @@ struct access::printer<vast::pattern>
 
   template <class Iterator>
   bool print(Iterator& out, const pattern& pat) const {
-    auto p = '/' << printers::str << ((pat.case_insensitive_) ? "/i" : "/");
+    auto escaper = detail::make_extra_print_escaper("/");
+    auto p = '/' << printers::escape(escaper)
+                 << ((pat.case_insensitive_) ? "/i" : "/");
```

Comment:
> **Note**
> This fixes the printing of patterns containing slashes. The pattern `/a\/b/` was printed as `/a/b/` before this change.

Reply:
Maybe this bug fix can be extracted into a separate `re2`-less pull request, just for the 3.0 release.